functests/10-test-template-validator-placement.sh

```diff
@@ -52,6 +52,35 @@ if [ "$AFFINITY" != "testKey" ]; then
   RET=1
 fi
 
+echo "[test_id:4976]: Check if patch is applied without error"
+oc patch -n ${TEST_NS} KubevirtTemplateValidator kubevirt-template-validator --type='json' -p='[{"op": "remove", "path": "/spec/affinity"}, {"op": "remove", "path": "/spec/nodeSelector"}, {"op": "remove", "path": "/spec/tolerations"}]'
+#wait until operator applies newest configuration
+sleep 60
```

Comment:
I think it is better to watch for a new `resourceVersion` on the CR instead of sleeping as this might be flakey